.github/workflows/test.yml

```
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches-ignore: []
```

actionlint (1.7.4)

"branches-ignore" section should not be empty

(syntax-check)